### PR TITLE
Implemented prebuilt tweet sentiment classifier [resolves #191]

### LINKED ISSUE #191: Twitter Sentiment Prebuilt Model
Build a prebuilt sentiment classifier on twitter data. Refer to template of  `news_classification` under `prebuilt`. 

Link to [Dataset](https://www.kaggle.com/enginkaraman/trke-twitter-duygu-analizi). It has a balanced distribution of targets and all three categories. Update if necessary with a better dataset until training. After that any updates on data would require splitting a new eval, data versioning, training and evaluating all existing models on that again.

### PR DESCRIPTION
**Pull request by ertugrul-dmr**
**`bblock/doc.py`:**

- Implemented same fix from #196

**`prebuilt/data`:** 

- gzipped twitter training data as `tweet_sentiment_train.csv.gz`.

**`prebuilt/twitter_sentiment.py`:**

- Implement empty_model, build and load methods for classification pipeline.
- Model parameters are selected using 3-fold cv. Optimizer code is not included.

**`tests/prebuilt/test_tweet_sentiment.py`**:

- Added assertion for testing cases.

Update `prebuilt/README.md` with usage and evaluation accuracy.

### CODE REVIEW

**Comment by husnusensoy** [comment]
Dataset seems to be pretty small. Can you send the citation to dataset used.

**Comment by ertugrul-dmr** [comment]
Link from given Issue #191 [here](https://www.kaggle.com/enginkaraman/trke-twitter-duygu-analizi). I'll be looking for similar datasets / papers for cases with more instances.
